<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dial_AsmDaq</class>
 <widget class="QDialog" name="Dial_AsmDaq">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Asm Daq Dialog Box</string>
  </property>
  <widget class="QCheckBox" name="checkBox_All">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>All</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm0">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm 0</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm 1</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm 2</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm 3</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm 4</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm 5</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm 6</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm 7</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm 8</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm 9</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm10">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm 10</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm11">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm 11</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Clear">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>161</width>
     <height>371</height>
    </rect>
   </property>
   <property name="title">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Do">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Do It</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Refresh">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Close">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>330</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Msg">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>371</width>
     <height>79</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="delayTrig">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>210</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="ModeDaq">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>211</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Mode Thor 3 mezzanines</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mode Thor Mezza0-Mezza1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mode Thor Mezza2-Mezza0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mode Thor Mezza2-Mezza1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mode Thor And 3 Mezza</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mode Thor And Mezza 0-2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mode Thor And Mezz 1-2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mode ASM Pied</string>
    </property>
   </item>
  </widget>
  <widget class="QRadioButton" name="choiceLogic">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>160</y>
     <width>102</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>T&amp;RIG AND</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_freqPied_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>210</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>*256*1Âµs</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_freqPied">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>210</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Freq Pied</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="FreqPied">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>210</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>40</number>
   </property>
  </widget>
  <widget class="QLabel" name="Label_delay_trig">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>190</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Delay Trigger</string>
   </property>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>checkBox_All</zorder>
  <zorder>checkBox_Asm0</zorder>
  <zorder>checkBox_Asm1</zorder>
  <zorder>checkBox_Asm2</zorder>
  <zorder>checkBox_Asm3</zorder>
  <zorder>checkBox_Asm4</zorder>
  <zorder>checkBox_Asm5</zorder>
  <zorder>checkBox_Asm6</zorder>
  <zorder>checkBox_Asm7</zorder>
  <zorder>checkBox_Asm8</zorder>
  <zorder>checkBox_Asm9</zorder>
  <zorder>checkBox_Asm10</zorder>
  <zorder>checkBox_Asm11</zorder>
  <zorder>checkBox_Clear</zorder>
  <zorder>pushButton_Do</zorder>
  <zorder>pushButton_Refresh</zorder>
  <zorder>pushButton_Close</zorder>
  <zorder>textEdit_Msg</zorder>
  <zorder>delayTrig</zorder>
  <zorder>ModeDaq</zorder>
  <zorder>choiceLogic</zorder>
  <zorder>Label_freqPied_2</zorder>
  <zorder>Label_freqPied</zorder>
  <zorder>FreqPied</zorder>
  <zorder>Label_delay_trig</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
